engine/src/main/java/org/terasology/engine/logic/characters/CharacterSystem.java

```diff
@@ -155,7 +155,7 @@ public String getDamageTypeName(Prefab damageType) {
             return damageType.getComponent(DisplayNameComponent.class).name;
         } else {
             String damageTypeName = damageType.getName();
-            logger.info("{} is missing a readable DisplayName", damageTypeName);
+            logger.atInfo().addArgument(damageTypeName).log("{} is missing a readable DisplayName");
```

Comment:
why do we need fluent API here? it's just a variable reference, not a method call
```suggestion
            logger.info("{} is missing a readable DisplayName", damageTypeName);
```